authentication/esignet-integration-impl/src/test/java/io/mosip/authentication/esignet/integration/service/IdaVCIssuancePluginImplTest.java

```diff
@@ -204,6 +203,38 @@ public void getVerifiableCredentialWithLinkedDataProof_withInValidIndividualId_t
         }
     }
 
+    @Test
+    public void getVerifiableCredentialWithLinkedDataProof_withInValidDetails_thenFail() throws Exception {
+
+        ReflectionTestUtils.setField(idaVCIssuancePlugin,"vciExchangeUrl","http://example.com");
+
+        VCRequestDto vcRequestDto = new VCRequestDto();
+        vcRequestDto.setFormat("ldp_vc");
+        vcRequestDto.setContext(Arrays.asList("context1","context2"));
+        vcRequestDto.setType(Arrays.asList("VerifiableCredential"));
+        vcRequestDto.setCredentialSubject(Map.of("subject1","subject1","subject2","subject2"));
+
+        OIDCTransaction oidcTransaction = new OIDCTransaction();
+        oidcTransaction.setIndividualId("individualId");
+        oidcTransaction.setKycToken("kycToken");
+        oidcTransaction.setAuthTransactionId("authTransactionId");
+        oidcTransaction.setRelyingPartyId("relyingPartyId");
+        oidcTransaction.setClaimsLocales(new String[]{"en-US", "en", "en-CA", "fr-FR", "fr-CA"});
+
+        IdaResponseWrapper<IdaVcExchangeResponse<JsonLDObject>> mockResponseWrapper = new IdaResponseWrapper<>();
+        mockResponseWrapper.setErrors(Collections.singletonList(new IdaError()));
```

Comment:
The test case should validate the custom error code returned from the plugin. So the IdaError should return with a valid custom error code instead of an empty constructor.

We should finally assert the custom error code being returned.

Reply:
added anusha